README.md

```diff
@@ -1,32 +1,167 @@
-# Switch Statement
+# Switch Statements
 
 ## Learning Goals
 
-- Learning Goal 1
-- Learning Goal 2
+- Explain what a switch statement is.
+- Discuss why a `break` statement should be included.
 
-## Introduction
+## What is a Switch Statement?
 
-A 1-2 sentence summary of what will be covered.
+A **switch statement** is a multiway branch that is more efficient than nested
+`if` statements in that it can have multiple execution paths. To help understand
+more about what goes into a switch statement, let us look at the syntax:
 
-## Topic 1
+```java
+switch (expression) {
+    // case statements
+    // values must be the same data type as the expression
+    case value1:
+        // statement sequence
+        break;    // breaks out of the case statement
+        
+    case value2:
+        // statement sequence
+        break;    // breaks out of the case statement
 
-Discuss the topic of the reading. Break into sections and sub-sections as
-appropriate to help with the reader's orientation with the material and future
-referencing.
+    // There can be multiple case statements
+    // and the default case statement is executed if none of the above
+    // cases are true
+    default:
+        // statement sequence
+        // no break is needed in the default case
+}
+```
 
-## Topic 2
+Duplicate cases are not allowed in switch statements.
 
-Discuss the topic of the reading. Break into sections and sub-sections as
-appropriate to help with the reader's orientation with the material and future
-referencing.
+The `break` statement, while optional, is highly recommended. It tells Java that
+they have finished writing the code needed within the case and that it can get
+out of the `switch`. When the code reaches a `break`, it will exit from the
+`switch` and continue on with the next statement outside the `switch`. More on
+what happens if we omit the `break` statement in a little bit.
 
-## Conclusion
+`default` statements are also optional and usually appear at the end of the
+`switch`. The `default` statement sequence is executed if no `case` matches. If
+the `default` statement is omitted, then no action will take place.
 
-A short one or two paragraph summary of the contents of the lessons, recapping
-the learning goals.
+The advantage of switch statements are that they tend to be more efficient than
```

Comment:
Are switch-statements more efficient in that the program will execute faster or use less memory?

Reply:
I believe there is a small performance enhancement, but I guess I just meant that `switch` statements are usually nicer to read; therefore helping readability and maintainability of the code.

I'll adjust this paragraph to reflect that.